cmd/Makefile.am

```diff
@@ -17,6 +17,10 @@ if USE_INTERNAL_BPF_HEADERS
 VENDOR_BPF_HEADERS_CFLAGS = -I$(srcdir)/libsnap-confine-private/bpf/vendor
 endif
 
+# only set permitted capabilities
+SNAP_CONFINE_CAPS = \
```

Comment:
Could we eliminate this list with `$(file <snap-confine/snap-confine.caps)` in automake?

Is this variable used anywhere?